solutions/cf-20C.mdx

```diff
@@ -76,4 +76,57 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+	
+```py
+
+import heapq
+import sys
+
+
+def dijkstra(graph, source):
+    vis = [False] * (len(graph) + 1)  # initializing each node as unvisited
+    dist = [sys.maxsize] * (len(graph) + 1)  # initializing distance of source to all vertices as +infinity
```

Comment:
use `float('inf')` instead of sys.maxsize and remove `import sys` while you're at it

Reply:
Getting verdict as "Memory limit exceeded" When I use float('inf') instead of sys.maxsize on codeforces

Using float('inf')
<img width="683" alt="image" src="https://user-images.githubusercontent.com/78244925/175800398-edc89430-7ca8-4510-9270-ecfe58bc393a.png">

Using sys.maxsize
<img width="585" alt="image" src="https://user-images.githubusercontent.com/78244925/175800418-25cb59a3-4259-4e4c-bd5b-82e967efc079.png">